src/Resources/app/storefront/src/js/plugin/nosto-analytics.js

```
@@ -51,17 +51,27 @@ export default class NostoAnalytics extends window.PluginBaseClass {
             if (!response.ok) {
                 console.error('Tracking failed, no response received.');
             }
+            return true;
         } catch (error) {
             console.error('Error sending tracking request:', error);
+            return false;
```

maybe its better to throw the error up as we have try/catch above this function call